package.json

```diff
@@ -25,14 +25,15 @@
     "dependencies": {
         "bcrypt": "^2.0.1",
         "body-parser": "^1.18.3",
+        "cors": "^2.8.4",
+        "dotenv": "^5.0.1",
         "errorhandler": "^1.5.0",
         "express": "^4.16.3",
         "joi": "^13.3.0",
         "jsonwebtoken": "^8.2.1",
         "morgan": "^1.9.0",
         "pg": "^7.0.3",
-        "winston": "^2.4.2",
-        "dotenv": "^5.0.1"
+        "winston": "^2.4.2"
```

Comment:
Where are you using this package?

Reply:
I used it previously but forgot to uninstall it